BoxContentSDK/BoxContentSDK/Operations/BOXAPIThumbnailDownloadDataOperation.m

```diff
@@ -0,0 +1,13 @@
+//
+//  BOXAPIThumbnailDownloadOperation.m
+//  BoxContentSDK
+//
+//  Created by Clement Rousselle on 3/16/16.
+//  Copyright © 2016 Box. All rights reserved.
+//
+
+#import "BOXAPIThumbnailDownloadDataOperation.h"
```

Comment:
Or if we buy into the 'small operation' not-only-thumbnails stance, we should make this isSmallDownload which would be applicable for thumbnails, metadata download, etc.